src/main.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+import { HashRouter } from 'react-router-dom';
+import { Provider } from 'react-redux';
+import App from './App';
+import store from './redux/configureStore';
+
+import './index.css';
+
+ReactDOM.createRoot(document.getElementById('root')).render(
+  <React.StrictMode>
+    <Provider store={store}>
+      <HashRouter>
+        <App />
```

Comment:
- [ ] [OPTIONAL] Your app doesn't work properly, kindly make sure every thing is working as expected, as for now, The app throws this error
![Screenshot_20230106_174743](https://user-images.githubusercontent.com/25227022/211059115-0d585e71-328f-44b1-bad5-49ad6aa18ffd.png)